torchao/float8/README.md

```diff
@@ -1,17 +1,14 @@
 # torchao.float8
 
-This is a workflow for accelerating training with float8 in native PyTorch
-according to the recipes laid out in https://arxiv.org/pdf/2209.05433.pdf.
-The codebase strives to stay small, easily hackable, debuggable with native PyTorch tooling,
+This is a workflow for accelerating training with [float8](https://arxiv.org/pdf/2209.05433.pdf) in native PyTorch.
+With ``torch.compile`` on, we demonstrate e2e pretraining throughput speedups of up to [**1.5x at 512 GPU / 405B parameter count scale**](https://pytorch.org/blog/training-using-float8-fsdp2/),
```

Comment:
Do you want to add the 2k gpu scale as well, via titan?

Reply:
I do, but we don't have a link for it yet.  We should definitely add it once there is a link!